tests/ert/unit_tests/run_models/test_status_events_serialization.py

```diff
@@ -0,0 +1,158 @@
+                        index="0",
+                        status=state.FORWARD_MODEL_STATE_FINISHED,
+                        name="fm_step_0",
+                        end_time=dt(2019, 1, 1),
+                    )
+                    .build(
+                        real_ids=["1"],
+                        status=state.REALIZATION_STATE_RUNNING,
+                    ),
+                    iteration_label="Foo",
+                    total_iterations=1,
+                    progress=0.5,
+                    realization_count=4,
+                    status_count={"Finished": 2, "Running": 1, "Unknown": 1},
+                    iteration=0,
+                ),
+                SnapshotUpdateEvent(
+                    snapshot=SnapshotBuilder(metadata=METADATA)
+                    .add_fm_step(
+                        fm_step_id="1",
+                        index="1",
+                        status=state.FORWARD_MODEL_STATE_FAILURE,
+                        name="fm_step_1",
+                    )
+                    .build(
+                        real_ids=["0"],
+                        status=state.REALIZATION_STATE_FAILED,
+                        end_time=dt(2019, 1, 1),
+                    ),
+                    iteration_label="Foo",
+                    total_iterations=1,
+                    progress=0.5,
+                    realization_count=4,
+                    status_count={"Finished": 2, "Failed": 1, "Unknown": 1},
+                    iteration=0,
+                ),
+                AnalysisStatusEvent(msg="hello"),
+                AnalysisTimeEvent(remaining_time=22.2, elapsed_time=200.42),
+                EndEvent(failed=False, msg=""),
+                RunModelStatusEvent(iteration=1, run_id=uuid.uuid1(), msg="Hello"),
+                RunModelTimeEvent(
+                    iteration=1,
+                    run_id=uuid.uuid1(),
+                    remaining_time=10.42,
+                    elapsed_time=100.42,
+                ),
+                RunModelUpdateBeginEvent(iteration=2, run_id=uuid.uuid1()),
+                RunModelDataEvent(
+                    iteration=1,
+                    run_id=uuid.uuid1(),
+                    name="Micky",
+                    data=DataSection(
+                        header=["Some", "string", "elements"],
+                        data=[["a", 1.1, "b"], ["c", 3]],
+                        extra={"a": "b", "c": "d"},
+                    ),
+                ),
+                RunModelUpdateEndEvent(
+                    iteration=3,
+                    run_id=uuid.uuid1(),
+                    data=DataSection(
+                        header=["Some", "string", "elements"],
+                        data=[["a", 1.1, "b"], ["c", 3]],
+                        extra={"a": "b", "c": "d"},
+                    ),
+                ),
+            ],
+        ),
+    ],
+)
+def test_status_event_serialization(events):
+    for event in events:
```

Comment:
One small comment could you move this to `parameterize`? If it fails for one of them it will give us a much better error message.

Reply:
Will do